generator/docs/GenerationTypes.md

```diff
@@ -18,6 +19,16 @@ Examples:
 Notes:
 - Random generation of data is infinite and is limited to 1000 by default, use `--max-rows` to enable generation of more data.
 
+## Full_sequential
```

Comment:
We should leave this feature as hidden for now - by design - as there are cases where it doesn't produce the expected set of results.